pkg/cli/admin/upgrade/status/examples/4.14.1-paused-worker-pool.detailed-output

```diff
@@ -1,6 +1,7 @@
 = Control Plane =
 Assessment:      Progressing
 Target Version:  4.14.1 (from 4.14.0)
+Waiting On:      N/A
```

Comment:
:thinking: 

Thoughts about hiding the line when N/A? "Nothing is updating right now" is already exposed through the "0 updating" in `Completion`.

Reply:
Alternatively, we could say something more informative, meaning "no operator is updating right now, so"

Reply:
Hided when N/A.
I was thinking of hiding it and then saw "N/A" used already in other places. Bad choice again.